### DISCUSSION TITLE
Getting the below error while compiling the C/C++ code in vscode

### DISCUSSION THREAD

**Discussion by apoorvsinghnegi** [microsoft/vscode-cpptools#9580] [General]
![1](https://user-images.githubusercontent.com/42599956/178921862-5b37481e-d642-4e93-bbd4-99f66d9587c4.PNG)
Can you suggest any changes. I have even put the correct environment

**Comment by Colengms** [comment]
Hi @apoorvsinghnegi .  `bits/stdc++.h` is a non-standard header provided by GCC.  Based on the path in your image, it looks like you're on Windows.  Have you installed a build of GCC for Windows (such as MinGW), and configured the C/C++ Extension to use it?  Or, if you're using MSVC to compile, perhaps just avoid including non-standard system headers.

**Reply by apoorvsinghnegi** [reply]
Hi @Colengms, I have already installed MinGW and I am also using appropriate C/C++ extensions

**Reply by Colengms** [reply]
Could you provide the contents of your `c_cpp_properties.json`?  Since that system header could not be found, it seems like your configuration might specify the MinGW compiler.  Also, could you provide the output of the `C/C++: Log Diagnostics` command, after the repro?
